frontend/lib/src/theme/emotionDarkTheme/themeColors.ts

```diff
@@ -23,10 +23,6 @@ export default {
   bgColor: colors.gray100,
   secondaryBg: colors.gray90,
   bodyText: colors.gray10,
-  navTextColor: colors.gray50,
-  navActiveTextColor: colors.gray10,
-  navIconColor: colors.gray70,
-  sidebarControlColor: colors.gray10,
```

Comment:
Removes sidebar nav-related properties from our semantic theme colors since they are now only adapted based on the text color and we are not planning to make those independently configurable